mypy/checkmember.py

```diff
@@ -245,6 +245,15 @@ def analyze_member_var_access(name: str, itype: Instance, info: TypeInfo,
                     getattr_type = expand_type_by_instance(bound_method, typ)
                     if isinstance(getattr_type, CallableType):
                         return getattr_type.ret_type
+        else:
+            setattr_func = info.get_method('__setattr__')
+            if setattr_func and setattr_func.info.fullname() != 'builtins.object':
+                setattr_f = function_type(setattr_func, builtin_type('builtins.function'))
```

Comment:
It's confusing to have separate variables called `setattr_func` and `setattr_f`. Perhaps the first one should be `setattr_method`.